tests/workers/test_base_worker.py

```diff
@@ -190,21 +192,20 @@ async def test_worker_sends_heartbeat_gets_id(experimental_logging_enabled, resp
 
         await worker.sync_with_backend()
 
-        assert worker.backend_id == test_worker_id
+        assert worker.worker_id == test_worker_id
```

Comment:
This worries me a bit since changing from `backend_id` to `worker_id` is technically a breaking change. I doubt it will break anyone, but could we keep `backend_id`?

Reply:
We introduce this as part of this process but it did get released, glad to change it back to be safe